### DISCUSSION TITLE
KEYBR.COM LINKEDIN ACCOUNT NOT FOUND

### DISCUSSION THREAD

**Discussion by KarthikVarma19** [aradzie/keybr.com#178] [Ideas]
PLEASE CREATE LINKEDIN ORGANIZATION ACCOUNT OF KEYBR.COM SO WE CAN TAG IT AND ALSO WE CAN PUT OUR PROFILE LINK OR CERTIFICATE LINK IN OUR CERTIFICATIONS OF OUR LINKEDIN PROFILE

**Comment by aradzie** [comment]
Hey, what is LinkedIn organization account?

**Reply by ajetz** [reply]
I think he just wants you to setup Keybr as an organization on Linkedin...
